app/javascript/tests/public_health/bulk_actions/UpdateAssignedUser.test.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import InfoTooltip from '../../../components/util/InfoTooltip';
+import UpdateAssignedUser from '../../../components/public_health/bulk_actions/UpdateAssignedUser';
+import { mockPatient1, mockPatient2, mockPatient6 } from '../../mocks/mockPatients';
+
+const onCloseMock = jest.fn();
+const mockToken = 'testMockTokenString12345';
+const mockAssignedUserDatalist = ['3682', '903874', '73892', '1689'];
+
+function getWrapper(patientsArray) {
+  let wrapper = shallow(<UpdateAssignedUser authenticity_token={mockToken} patients={patientsArray} close={onCloseMock} assigned_users={mockAssignedUserDatalist} />);
+  return wrapper;
+}
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('UpdateAssignedUser', () => {
+  it('Sets intial state after mount correctly', () => {
+    const wrapper = getWrapper([mockPatient1, mockPatient2]);
+    // componentDidMount is called when mounted so a timeout is needed to give it time to resolve.
+    setTimeout(() => {
```

Comment:
Don't love the timeout here... I'm pretty sure there is a way to check something once the component has mounted but I can't find it offhand right now

Reply:
also might be worth checking what the state is before the code in `componentDidMount` is executed